src/views/states/list/components/InterfacesTypeSection.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+
+import { ExpandableSectionToggle, List, ListItem, Tooltip } from '@patternfly/react-core';
+import { Td, Tr } from '@patternfly/react-table';
+import { NodeNetworkConfigurationInterface } from '@types';
+
+import './interfaces-table.scss';
+
+interface InterfacesTypeSectionProps {
+  interfaces: NodeNetworkConfigurationInterface[];
+  interfaceType: string;
+}
+
+const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = ({ interfaceType, interfaces }) => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
+
+  return (
+    <>
+      <Tr className="interfaces-table__interface-type-row">
+        <Td colSpan={6}>
+          <ExpandableSectionToggle isExpanded={isExpanded} onToggle={setIsExpanded}>
+            <strong className="pf-u-color-100">{interfaceType}</strong>
+          </ExpandableSectionToggle>
+        </Td>
+      </Tr>
+
+      {interfaces.map((iface) => {
+        const address = iface.ipv4?.address || iface.ipv6?.address;
+
+        return (
+          <Tr key={iface.name} isExpanded={isExpanded}>
+            <Td className="pf-m-width-20">{iface.name}</Td>
+            <Td>
+              {address ? (
+                <>
+                  {address?.[0]?.ip}/{address?.[0]?.['prefix-length']}
+                </>
+              ) : (
+                '-'
+              )}
+            </Td>
+            <Td>
+              {iface?.bridge?.port?.length ? (
```

Comment:
You are using `iface?` here, while just a few lines above you are not using "?" with `iface`. What is the reason for this difference?

Reply:
removed the first `?`